doc/client_lua_api.md

```diff
@@ -1152,6 +1152,10 @@ Methods:
     * See [`HUD definition`](#hud-definition-hud_add-hud_get)
 * `hud_get(id)`
     * returns the [`definition`](#hud-definition-hud_add-hud_get) of the HUD with that ID number or `nil`, if non-existent.
+* `hud_get_elements()`:
+    * returns a table indexed by all present HUD IDs and containing corresponding HUD element definition structures.
```

Comment:
```suggestion
    * Returns a table in the form `{ [id] = HUD definition, [id] = ... }`.
```
Shorter and clearly describes what the key and values are, respectively.

Reply:
Changed